app/assets/stylesheets/application.scss

```diff
@@ -63,7 +64,22 @@ h1 {
     border-color: black;
 }
 
-.btn {
-    color: black;
+
+
+.btn-primar {
```

Comment:
Yeea I had the feeling that this would be an issue because the name sounds incomplete. Thank you for mentioning that defining my own `btn-primary` is ok,  I thought I would have issues with bootstrap's already defined class.